docs/documentation/stories/continuous-integration.md

```diff
@@ -76,28 +79,23 @@ jobs:
   build:
     working_directory: ~/my-project
     docker:
-      - image: circleci/node:8-browsers
```

Comment:
When you provide only the major version, docker will always choose the most recent version.
E.g. `image: circleci/node:10-browsers` and then running `node -v` yields: `v10.6.0`

I would recommend reverting this change back to `- image: circleci/node:8-browsers`

Reply:
Thanks @markgoho, good point!